ComSemApp/student/views.py

```diff
@@ -718,8 +718,8 @@ def get_context_data(self, **kwargs) -> dict[str, Any]:
         return context
 
 practice_data = [
-                    {'transcription1':"This is a sentence transcription.",'accuracy1':50,'fluency1':75},
-                    {'transcription1':"This is a second sentence transcription.",'accuracy1':90,'fluency1':70}
+                    {'id':1,'transcription1':"This is a sentence transcription.",'accuracy1':50,'fluency1':75, 'correct':'This is the correct sentence.'},
```

Comment:
Why do transcription, accuracy, and fluency values have a '1' appended onto the end of their names?